### ISSUE TITLE
I want to make my iphone's dng image to sRGB image.

### ISSUE THREAD

**Issue by spnova12** [AbdoKamel/simple-camera-pipeline#1]
I made the dng file using the iraw app(https://itunes.apple.com/in/app/iraw/id917507085?mt=8)
But my dng file does not work in your code.
What are some possible solutions to this problem?
I'd like to ask you to give me some advice on this problem.
Thank you.

**Comment by AbdoKamel** [comment]
I found out that iPhone's DNGs are non-linearly compressed, which Matlab's Tiff library can't read.
One workaround would be to uncompress (linearize) the DNG file with Adobe's DNG Converter:
https://helpx.adobe.com/ca/photoshop/using/adobe-dng-converter.html

Just make sure to choose the "Uncompressed" option. 

See here for more details: https://helpx.adobe.com/ca/photoshop/using/adobe-dng-converter.html

I will try to implement this in the pipeline in the near future.

**Comment by spnova12** [comment]
Is not this the isp code actually used in your paper?
Did you use another isp to create the ground truth?
Thank you very much for your reply.

**Comment by AbdoKamel** [comment]
Yes, this is the code used in the paper. The issue with the compressed iPhone DNGs was resolved by the Adobe DNG Converter as mentioned. Hope this helps!

**Comment by AbdoKamel** [comment]
@spnova12  I forgot to ask, is your image already compressed? If you can share with me an example image of yours, I can try to figure out the exact issue.

**Comment by spnova12** [comment]
I am really grateful, but I do not use iphone anymore. Thank you for your kindness.
